src/libaktualizr/crypto/crypto.cc

```diff
@@ -234,8 +234,14 @@ bool Crypto::ED25519Verify(const std::string &public_key, const std::string &sig
                                      reinterpret_cast<const unsigned char *>(public_key.c_str())) == 0;
 }
 
-bool Crypto::parseP12(BIO *p12_bio, const std::string &p12_password, std::string *out_pkey, std::string *out_cert,
-                      std::string *out_ca) {
+std::string Crypto::bioToString(BIO *b) {
+  char *buf;
+  auto len = BIO_get_mem_data(b, &buf);
+  return std::string(buf, static_cast<size_t>(len));
+}
+
+bool Crypto::parseP12(BIO *p12_bio, const std::string &p12_password, std::string &out_pkey, std::string &out_cert,
```

Comment:
These were pointers because it's recommended by the google style guide: https://google.github.io/styleguide/cppguide.html#Output_Parameters.

We follow it pretty loosely though.

Reply:
This probably can (should?) be split into separate commit as it's orthogonal to CA bundling question.